core/src/main/kotlin/links/DRI.kt

```diff
@@ -94,10 +94,11 @@ data class TypeReference(val classNames: String, val typeBounds: List<TypeRefere
             TypeReference("", tp.upperBounds.map { from(it) })
 
         private fun from(t: KotlinType): TypeReference =
-            when (val d = t.constructor.declarationDescriptor) {
-                is TypeParameterDescriptor -> from(d)
-                else -> TypeReference(t.constructorName.orEmpty(), t.arguments.map { from(it) })
-            }
+            TypeReference(t.constructorName.orEmpty(), t.arguments.map { from(it) })
+//            when (val d = t.constructor.declarationDescriptor) {
+//                is TypeParameterDescriptor -> TypeReference("todo:${d.name}")
+//                else -> TypeReference(t.constructorName.orEmpty(), t.arguments.map { from(it) })
+//            }
```

Comment:
Please try not to leave commented code if it's not necessary.

Reply:
Removed. I've left it as a reminder that it's not permanent solution.